src/pages/loginForm.js

```diff
@@ -0,0 +1,108 @@
+import { Form, Input, Button } from 'antd';
+import { UserOutlined, LockOutlined } from '@ant-design/icons';
+import { Link, useNavigate } from 'react-router-dom';
+import '../styles/loginPage.scss';
+import { routes } from '../config/routes';
+import { httpsService } from '../utils/https.service';
+import { storageService } from '../utils/storage.service';
+
+
+export const LoginForm = () => {
+    const navigate = useNavigate();
+   
+
+    const handleLogin = async (body) => {       
+      
+      try { 
+        const response = await httpsService.post('/user/login', body);
+        
+        //console.log(response);
+        
+        if (response) {
```

Comment:
why do we need if statement here? Do we have a case when we have 200 status code and an empty response?

Reply:
if axios rejected with 401 error I'd have undefined in response handleLogin. So I need to check response to prevent undefined assigning to localStorage.

Reply:
I deleted it, cuz we have default header:  Content-Type: application/json; charset=utf-8